lighthouse-cli/bin.ts

```diff
@@ -16,14 +16,12 @@
  */
 'use strict';
 
-const _SIGINT = 'SIGINT';
-const _SIGINT_EXIT_CODE = 130;
 const _RUNTIME_ERROR_CODE = 1;
 const _PROTOCOL_TIMEOUT_EXIT_CODE = 67;
 
 const assetSaver = require('../lighthouse-core/lib/asset-saver.js');
 const getFilenamePrefix = require('../lighthouse-core/lib/file-namer.js').getFilenamePrefix;
-import {ChromeLauncher} from '../chrome-launcher/chrome-launcher';
+import {launch, LaunchedChrome} from '../chrome-launcher/chrome-launcher';
```

Comment:
We should avoid two ways to launch: `launch()` and `ChromeLauncher.launch()`. Can we consolidate to one? That may get confusing for users, especially with similar names.